src/main/java/org/tikv/common/log/SlowLogSpanImpl.java

```diff
@@ -26,21 +26,31 @@
   private final String name;
   private long startMS;
   private long endMS;
+  /**
+   * use System.nanoTime() to calculate duration, cause System.currentTimeMillis() is not monotonic
+   */
+  private long startNano;
+
+  private long endNano;
 
   public SlowLogSpanImpl(String name) {
     this.name = name;
     this.startMS = 0;
+    this.startNano = 0;
     this.endMS = 0;
+    this.endNano = 0;
   }
 
   @Override
   public void start() {
     this.startMS = System.currentTimeMillis();
```

Comment:
I think we can remove the previous startMS and switch to nanoTime

Reply:
we will lose the time point information if removed.
` System.nanoTime()` can be only used to measure the difference of time.

Reply:
how about we use a specialized clock to calculate the difference of time more accurately without checking currentMillis() every time? I can optimize it after your PR is merged.

Reply:
fixed